zipline/assets/_assets.pyx

```diff
@@ -1,3 +1,4 @@
+# cython: embedsignature=True
```

Comment:
This makes Cython include the signature as a string in the generated C source so that inspection tools (e.g. sphinx, or IPython) can examine it).